test/periodic.jl

```diff
@@ -2,47 +2,32 @@
     @testset "_periodic" begin
         @testset "$f" for f in (sin, cos)
             # A typical 24-hour day
-            for h in 0:24
-                result = _periodic(
-                    f,
-                    ZonedDateTime(2015, 6, 1, tz"America/Winnipeg") + Hour(h),
-                    Day(1),
-                )
-                expected = f(2π / 24 * h)
-                @test result ≈ expected atol=1e-14
-            end
+            x = ZonedDateTime(2015, 6, 1, tz"America/Winnipeg") .+ collect(Hour.(0:2))
```

Comment:
can we give these more meaningful names other than `x`? Like `normal_day`, `spring_dst`, `fall_dst`?

Reply:
also might help to put them in their own `@testsets`